pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -333,6 +338,21 @@ stdenv.mkDerivation {
   postInstall = ''
     sed -i "s|{exec_prefix}/$out|{exec_prefix}|;s|{prefix}/$out|{prefix}|" \
       "$out/lib/pkgconfig/opencv4.pc"
+  ''
+  # install python distribution information, so other packages can `import opencv`
+  + lib.optionalString enablePython ''
+    pushd $NIX_BUILD_TOP/$sourceRoot/modules/python/package
```

Comment:
I have seen other packages that are not properly packaged for python, maybe can you extract it in a setup hook?

Reply:
@pasqui23 Can you please name some?

Reply:
Ok my bad I swear that z3 too, but checking closely it does not look to have those problems

Reply:
We can always get back to this is an issue